src/backend/mailbox_manager/models.py

```diff
@@ -223,9 +223,5 @@ def save(self, *args, **kwargs):
             raise exceptions.ValidationError(
                 _("You can't create a mailbox for a disabled domain.")
             )
-
-        if self._state.adding:
-            return super().save(*args, **kwargs)
-
-        # Update is not implemented for now
-        raise NotImplementedError()
+        # todo: allow local_part modification only for a pending mailbox
+        return super().save(*args, **kwargs)
```

Comment:
Please update the method docstring accordingly ;)

Reply:
Done. Thanks